.github/workflows/download_test.yml

```diff
@@ -0,0 +1,34 @@
+name: "Integration Download test"
+# NOTE: This runs only integration tests that download data,
+# as denoted by the pytest downloads_data mark.
+
+on:
+  push:
+    branches:
+      - "main"  # Releases
+      - "development"  # Feature PR merges
+
+jobs:
+  test:
+    name: "Download Data Tests"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - uses: "actions/checkout@v4"
+
+      - uses: "./.github/actions/install-icepyx"
+        with:
+          python-version: "3.12"
+
+      - name: "Run download tests"
+        env:
+          EARTHDATA_LOGIN: "icepyx_devteam"
```

Comment:
Is this envvar needed? We only pass the other two into the other integration tests.

Reply:
I think @betolink added it in to the integration action during his harmony integration work, so I included it here figuring then they'd match...